docs/INTEGRATION_RECOMMENDATIONS.md

```diff
@@ -20,7 +20,8 @@ Context variables may be applied to individual recommendation profiles similar t
 | options.siteId | global siteId overwrite | all | optional global siteId overwrite |
 | options.categories | category path | all | optional category identifier to restrict recommendations |
 | options.branch | template branch overwrite | all | optional branch overwrite for recommendations template (advanced usage) |
-| options.batched | boolean (default: `true`)| all | only applies to recommendation context, optional disable profile from being batched in a single request, can also be set globally [via config](https://github.com/searchspring/snap/tree/main/packages/snap-controller/src/Recommendation) |
+| options.batched | boolean (default: `true`)| all | only applies to recommendation context, optional disable profile from being batched in a single request, can also be set globally [via config](https://github.com/searchspring/snap/tree/main/packages/snap-controller/src/Recommendation) | 
+| options.order | number | all | optional order number for recommendation params to be added to the batched request. `undefined` will be sent to back. 
```

Comment:
> `undefined` will be sent to back.

Consider rephrasing to something like:

Profiles that do not specify an order will be placed at the end in the occurrence they appear in the DOM.